World/Dungeon/Dungeon.gd

```diff
@@ -39,23 +39,22 @@ func _on_ProceduralMazeLevel_addEntrance(position: Vector2):
 	var entrance = mapMaker.entranceScene.instance() as DungeonExit
 	entrance.position = position
 	var connectionResults = entrance.connect("exit_dungeon", self, "_on_exit_dungeon")
-	
 	entranceContainer.add_child(entrance)
-	
+
 func _on_exit_dungeon(body):
 	print("popup exit confirmation")
 	confirm_exit();
 
 func confirm_exit():
 	print("show exit dungeon confirmation")
-	$CanvasLayer/ConfirmExit.popup_centered()
+	$CanvasLayer/ConfirmExit.popup_centered()  # FIXME doesn't pop up
```

Comment:
fwe'll need to change how this works anyway otherwise it'll pop for everyone at the same time.

I think we'll need to make a general "dialog" on the player canvas that can accept inputs for the wording, and then this function would need to subscribe to the confirmation and cancel events to handle the player response contextually.